designdocs/capacity_staking_rewards_economic_model.md

```diff
@@ -0,0 +1,110 @@
+# Capacity Staking Rewards Economic Model
+
+
+## Context and Scope:
+The Frequency Transaction Payment system uses Capacity to pay for certain transactions on chain.  Accounts that wish to pay with Capacity must:
+1. Have an [MSA](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/accounts.md)
+2. Be a [Provider](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_registration.md) (see also [Provider Permissions and Grants](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/provider_permissions.md))
+3. Stake a minimum amount of FRQCY token to receive [Capacity](https://github.com/LibertyDSNP/frequency/blob/main/designdocs/capacity.md).
+
+There is also a business case for allowing any token holder to lock up its tokens in exchange for a reward - known as _staking_ - while also targeting a Provider to receive some Capacity.
+
+[//]: # (A short description of the landscape in which the new system is being built, what is actually being built.)
+[//]: # (It may also say what is not being built, and any assumptions.)
+[//]: # (Example: The proposed feature is a testing library. The context is: the library is for our chosen blockchain. The scope is: this is for a specific repository, so it's not meant to be reused. That means it won't be a separate package, and the code will be tailored for this repo. One might also say that the scope is also limited to developer testing, so it's not meant to be used in CI or a test environment such as a test blockchain network.)
+
+## Problem Statement:
+A system consisting only of providers and coinless users who delegate to providers will tend toward centralization.
+To build a self-sustaining Frequency network where control is decentralized, a variety of economic solutions are needed.  One of these is the ability to stake FRQCY token in return for something; this creates an incentive for participation and involvement with the chain fundamentals and governance.
+
+How is that so?  Capacity is how Frequency intends Providers to pay for the vast majority of their on-chain messages. If Providers receive some capacity when end users stake, then they lose capacity if those end users unstake. If a Provider's Capacity from Reward staking is significant, this gives Reward Stakers some power over Providers. If a Provider is utilizing all or nearly all their Capacity almost every Epoch -- which they should be if trying to be economical --  then even a small percentage of lost Capacity will literally cost them to replace it.
```

Comment:
I like this sentence:
> If a Provider's Capacity from Reward staking is significant, this gives Reward Stakers some power over Providers. 
I would add to it  - because their ability to operate **efficiently** depends on this capacity.